tools/syz-env

```diff
@@ -82,7 +82,7 @@ if [ ! -z "$SYZ_ENV_BUILD" ]; then
 	docker build "$SCRIPT_DIR/docker/$IMAGE" --tag "$IMAGE_NAME" ${BUILDARGS[@]}
 else
 	IMAGE_NAME="gcr.io/syzkaller/$IMAGE"
-	docker pull -q "$IMAGE_NAME"
+#	docker pull -q "$IMAGE_NAME"
```

Comment:
Why?

Reply:
Ah, that was committed by mistake. Thanks for noticing!